extensions/ql-vscode/src/view/data-extensions-editor/MethodRow.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  ExternalApiUsage,
+  ModeledMethod,
+} from "../../data-extensions-editor/interface";
+import {
+  VSCodeDataGridCell,
+  VSCodeDataGridRow,
+  VSCodeDropdown,
+  VSCodeOption,
+  VSCodeTextField,
+} from "@vscode/webview-ui-toolkit/react";
+import * as React from "react";
+import { useCallback, useMemo } from "react";
+import styled from "styled-components";
+
+const Dropdown = styled(VSCodeDropdown)`
+  width: 100%;
+`;
+
+const TextField = styled(VSCodeTextField)`
+  width: 100%;
+`;
+
+type SupportedUnsupportedSpanProps = {
+  supported: boolean;
+};
+
+const SupportedUnsupportedSpan = styled.span<SupportedUnsupportedSpanProps>`
+  color: ${(props) => (props.supported ? "green" : "red")};
+`;
+
+type Props = {
+  method: ExternalApiUsage;
+  model: ModeledMethod | undefined;
+  onChange: (method: ExternalApiUsage, model: ModeledMethod) => void;
```

Comment:
Should this be 

```suggestion
  onChange: (apiUsage: ExternalApiUsage, modeledMethod: ModeledMethod) => void;
```

... or something like that?